WordPress/Classes/ViewRelated/Comments/CommentsViewController.m

```diff
@@ -165,8 +206,15 @@ - (CGFloat)tableView:(UITableView *)tableView heightForRowAtIndexPath:(NSIndexPa
 
 - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(NSIndexPath *)indexPath
 {
+    if ([self usesUnifiedList]) {
+        ListTableViewCell *listCell = (ListTableViewCell *)[tableView dequeueReusableCellWithIdentifier:ListTableViewCell.reuseIdentifier
```

Comment:
Since I'm using `dequeueReusableCellWithIdentifier:forIndexPath:`, the method guarantees that it returns a non-null instance of the cell, as long as it's properly registered:

```objc
// newer dequeue method guarantees a cell is returned and resized properly, assuming identifier is registered
- (__kindof UITableViewCell *)dequeueReusableCellWithIdentifier:(NSString *)identifier forIndexPath:(NSIndexPath *)indexPath API_AVAILABLE(ios(6.0));
```

I think it's unnecessary to add additional `if (!cell)` checks, but I'd like to double check this again with you 😄 . If you think we still need the check, I'll add it! 🙂

Reply:
> as long as it's properly registered

Ah, this part I did not know. Reread the documentation for that method, and now I do. 😄 I was distracted by the old code. Sorry for the bother!